src/Code/CodeGroup.tsx

```diff
@@ -18,7 +18,6 @@ export type CodeGroupPropsBase = {
   /**
    * Background color for the tooltip saying `Copied` when clicking the clipboard button.
```

Comment:
You should delete the comments above the prop too.

Reply:
Removed comments for the prop in codeblock and codegroup component